prefect-v1-legacy/devops/monitoring/README.md

```diff
@@ -0,0 +1,345 @@
+
+The implementation looks like this:
+
+![Document systems (2).png](https://github.com/PrefectHQ/prefect-recipes/blob/prom-monitoring/monitoring/imgs/Diagram.png)
+
+## Requirements
+
+To successfully implement and configure the Prefect-Prometheus-Exporter, the following are required:
+
+- [Prometheus-Operator](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-prometheus-stack)
+- Prefect Server / Prefect Cloud endpoint is accessible
+- Helm
+- Public / Private Container Registry
+- Docker
+
+This document will focus on installation and configuration on the Exporter specifically, as this is being introduced to the architecture / system. 
+
+Grafana and Prometheus should already have at least a default installation in operational environment, but do not need to be in-cluster. 
+
+It is also assumed there is a currently accessible Prefect configuration. This document and configuration is written solely around Prefect Apollo (Prefect 1.0) which relies on GraphQL. 
+This work will be re-factored to support Prefect Orion (Prefect 2.0) which transitions to a RESTful API.
+
+## Components
+
+1. **Prefect Server**: depicted above, the entry point for requests is the Apollo pod, by default on `:4200`. 
+2. **Prefect** **Database:** where all the information is stored - this data is retrieved by requests from Apollo through Hasura/GraphQL. Default implementations are either SQLite or Postgres.
+3. **Prefect**-**Prometheus**-**Exporter:** This is the container that will be built and inserted into the cluster. It is functionally a proxy - it retrieves information from the database through queries to Apollo, and exposes those metrics by default on `:8000/metrics`.
+4. **Prometheus:** This is the time-series database that aggregates and scrapes metrics in a ‘pull’ model. Each ‘target’ that it scrapes is configured through a ServiceMonitor (detailed below). A ServiceMonitor is a CRD (Custom Resource Definition) - it is not native to Kubernetes, but created as part of the Prometheus-Operator installation. It reaches targets on the port specified in their ServiceMonitor manifest. It is exposed by default on `:9090`.
+5. **Grafana:** This is the visualization service and dashboarding. Grafana is configured with Prometheus as a data-source. By default it is accessible on `:3000`.
+
+## Source Repository
+
+The source location for this content is [PrefectHQ/prefect-recipes](https://github.com/prefecthq/prefect-recipes).
+At the time of writing this, it exists in the branch [prom-monitoring](https://github.com/PrefectHQ/prefect-recipes/tree/prom-monitoring/monitoring).
```

Comment:
I assume this branch is going to get outdated as we update recipes, can we remove this?

Reply:
Removed, thanks!